src/compose.ts

```diff
@@ -493,7 +493,6 @@ export interface ComposeFactory<T, O extends Options> extends ComposeMixinable<T
 	/**
 	 * Extend the factory prototype with the supplied object literal, class, or factory
 	 *
-	 * @deprecated
```

Comment:
Gotcha, I suspected as much. For adding a mixin, we would still use the `.mixin` API, such as `createMyClass.mixin(createMyMixin)`? Or would there be a new API that would be recommended for that?